### DISCUSSION TITLE
Does regenerating a PAT expire the old?

### DISCUSSION THREAD

**Discussion by sss-ng** [community/community#142976] [Repositories] [answered]
### GitHub PAT
In other words, does the 'regenerate' button just creating new tokens that are also valid, or is it invalidating the old one, and then creating a new one. 

### Body

I believe the answer to this is 'yes', but I wanted to check with some other users since I can't find anything explicit in the docs about this, and I want to be 100% sure that the old tokens are no longer valid.

**Comment by aymena80s545** [comment]
https://t.me/Helloo_FRIENDD

**Comment by sss-ng** [comment]
I just tested this myself, and it seems that regenerating a token does cause the previous to expire 👍

**Comment by davevad93** [comment, marked as answer]
Hi @sss-ng ,

Regenerating a new token means creating a new token with the same permissions as the old one, the old token is deactivated so any application that uses it will need to be updated with the newly regenerated token.

**Comment by kingmaven** [comment]
Hi @davevad93 can I regenerate an equivalent token?
